packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -9,28 +9,26 @@ import { SafeCall } from "../libraries/SafeCall.sol";
 import { IOptimismMintableERC20, ILegacyMintableERC20 } from "./IOptimismMintableERC20.sol";
 import { CrossDomainMessenger } from "./CrossDomainMessenger.sol";
 import { OptimismMintableERC20 } from "./OptimismMintableERC20.sol";
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 
 /// @custom:upgradeable
 /// @title StandardBridge
 /// @notice StandardBridge is a base contract for the L1 and L2 standard ERC20 bridges. It handles
 ///         the core bridging logic, including escrowing tokens that are native to the local chain
 ///         and minting/burning tokens that are native to the remote chain.
-abstract contract StandardBridge {
+abstract contract StandardBridge is Initializable {
     using SafeERC20 for IERC20;
 
     /// @notice The L2 gas limit set when eth is depoisited using the receive() function.
     uint32 internal constant RECEIVE_DEFAULT_GAS_LIMIT = 200_000;
 
-    /// @notice Messenger contract on this domain.
-    CrossDomainMessenger public immutable MESSENGER;
-
     /// @notice Corresponding bridge on the other domain.
-    StandardBridge public immutable OTHER_BRIDGE;
+    StandardBridge internal immutable _OTHER_BRIDGE;
```

Comment:
Should this have a `@custom:network-specific` natspec tag?

Reply:
Yes, good catch